downstream/modules/platform/proc-controller-create-container-group.adoc

```diff
@@ -53,7 +53,7 @@ oc get secret $SA_SECRET -o json | jq '.data["ca.crt"]' | xargs | base64 --decod
 . Use the contents of `containergroup-sa.token` and `containergroup-ca.crt` to provide the information for the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_user_guide/index#ref-controller-credential-openShift[OpenShift or Kubernetes API Bearer Token] required for the container group.
 
 To create a container group, create an link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_user_guide/index#ref-controller-credential-openShift[OpenShift or Kubernetes API Bearer Token] credential to use with your container group.
-For more information, see link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_user_guide/index#controller-getting-started-create-credential[Creating a credential] in the _{ControllerUG}_.
+For more information, see link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_user_guide/index#controller-create-credential[Creating a credential] in the _{ControllerUG}_.
```

Comment:
```suggestion
For more information, see link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_user_guide/index#controller-create-credential[Creating a credential] in _{ControllerUG}_.
```
Deleting "the" because it renders as "the Using automation execution" in the built doc